ExilenceNextApp/src/index.tsx

```diff
@@ -49,7 +49,7 @@ enableLogging({
 
 configure({ enforceActions: 'observed' });
 
-moment.locale(electronService.remote.app.getLocale());
+moment.locale(electronService.app.getLocale());
```

Comment:
This seems to crash locally

Reply:
Yeah, the `remote` is depracated so need some alternative to that 🤔

Reply:
Oof, interesting..